authors/apps/articles/models.py

```diff
@@ -19,3 +19,19 @@ class Article(models.Model):
 
     def __str__(self):
         return self.title
+
+
+class LikeArticle(models.Model):
+    LIKE = 'like'
```

Comment:
What are you trying to achieve here???

Reply:
I was creating the choices to be passed, So its like true or false but i then used like or dislike